flusher.go

```diff
@@ -375,3 +383,115 @@ func (s *Server) flushTraces(ctx context.Context) {
 		w.Flush()
 	}
 }
+
+// forwardGRPC forwards all input metrics to a downstream Veneur, over gRPC.
+func (s *Server) forwardGRPC(ctx context.Context, wms []WorkerMetrics) {
+	span, _ := trace.StartSpanFromContext(ctx, "")
+	span.SetTag("protocol", "grpc")
+	defer span.ClientFinish(s.TraceClient)
+
+	exportStart := time.Now()
+	metrics := s.exportForwardMetrics(wms)
+	span.Add(
+		ssf.Timing("forward.duration_ns", time.Since(exportStart),
+			time.Nanosecond, map[string]string{"part": "export"}),
+		ssf.Gauge("forward.post_metrics_total", float32(len(metrics)), nil),
+	)
+
+	if len(metrics) == 0 {
+		log.Debug("Nothing to forward, skipping.")
+		return
+	}
+
+	entry := log.WithFields(logrus.Fields{
+		"metrics":     len(metrics),
+		"destination": s.ForwardAddr,
+		"protocol":    "grpc",
+	})
+
+	conn, err := grpc.Dial(s.ForwardAddr, grpc.WithInsecure())
```

Comment:
This is setting up the gRPC connection at send time. The expected gRPC pattern, AIUI, is to establish (`Dial()`) the underlying connection ahead of time, then repeatedly reuse it when it comes time to send. gRPC handles maintaining connection state under the hood.

So we'll need to extend `Server` to hold on to the `grpc.ClientConn` permanently, then each call to this `forwardRPC()` method will just create a `NewForwardClient()` from that existing conn.

Moving connection establishment to a startup routine may also make it a little easier to pull in config options so that `grpc.WithInsecure()` doesn't have to be hardcoded.